services/portal/values-idfdev.yaml

```diff
@@ -4,7 +4,11 @@ resources:
   limits:
     memory: "2Gi"
 
+image:
+  tag: "suit-2022.3"
```

Comment:
I think this tag has already been used.  This should be advanced to 2022.4.

Reply:
Also, the internal version reporting in `app.config` needs to be updated accordingly.

Reply:
yes, we can make it 2022.4

Reply:
See https://github.com/lsst/suit/pull/41 - it would also be OK for the new version to be 2022.3.1, but we should